src/components/Video/Video.tsx

```diff
@@ -144,77 +92,19 @@ const Video: React.FC<Props> = ({
-                label={hasShared ? t('video:copied_url') : t('video:share')}
-                startIcon={hasShared ? <Check /> : <Share />}
-                onClick={onShareClick}
-                active={hasShared}
+                color={isFavorite ? 'primary' : 'default'}
                 fullWidth={breakpoint < Breakpoint.md}
               />
             )}
+            {shareButton}
           </div>
         </div>
-        <div className={classNames(styles.poster, styles[poster])} style={{ backgroundImage: `url('${posterImage}')` }} />
+        <div className={classNames(styles.poster, styles[posterMode])} style={{ backgroundImage: `url('${poster}')` }} />
       </div>
       {!!children && <div className={classNames(styles.related, styles.mainPadding)}>{children}</div>}
-      <Fade open={play}>
-        <div className={styles.playerContainer}>
-          <div className={styles.player}>
-            <Cinema
-              item={item}
-              feedId={feedId}
-              onPlay={handlePlay}
-              onPause={handlePause}
-              onComplete={handleComplete}
-              onUserActive={handleUserActive}
-              onUserInActive={handleUserInactive}
-            />
-          </div>
-          <Fade open={!isPlaying || userActive}>
-            <div className={styles.playerOverlay}>
-              <div className={styles.playerContent}>
-                <IconButton aria-label={t('common:back')} onClick={goBack} className={styles.backButton}>
-                  <ArrowLeft />
-                </IconButton>
-                <div>
-                  <h2 className={styles.title}>{title}</h2>
-                  <div className={styles.metaContainer}>
-                    {isSeries && <div className={classNames(styles.seriesMeta, styles.seriesMetaPlayer)}>{seriesMeta}</div>}
-                    <div className={styles.meta}>{metaString}</div>
-                  </div>
-                </div>
-              </div>
-            </div>
-          </Fade>
-        </div>
-      </Fade>
-      <Alert open={warning !== null} message={warning} onClose={clearWarning} />
-      {!!trailerItem && (
-        <Modal open={playTrailer} onClose={onTrailerClose}>
-          <div className={styles.trailerModal}>
-            <Cinema
-              item={trailerItem}
-              onPlay={handlePlay}
-              onPause={handlePause}
-              onComplete={onTrailerClose}
-              onUserActive={handleUserActive}
-              onUserInActive={handleUserInactive}
-              isTrailer
-            />
-            <div className={classNames(styles.playerOverlay, { [styles.hidden]: isPlaying && !userActive })} />
-            <div className={classNames(styles.trailerMeta, styles.title, { [styles.hidden]: isPlaying && !userActive })}>{`${title} - Trailer`}</div>
-            <ModalCloseButton onClick={onTrailerClose} visible={!isPlaying || userActive} />
-          </div>
-        </Modal>
-      )}
     </div>
   );
 };
```

Comment:
Would it make sense to rename `Video` to something like `MediaDetails` / `VideoDetails` / `VideoOverview` / etc., since there is no video or player in it now?

Reply:
Agree! I've renamed it to `<VideoDetails />`